pages/common/cargo-logout.md

```diff
@@ -0,0 +1,40 @@
+# cargo logout
+
+> Remove an API token from the registry locally.
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-logout.html>.
+
+- This command will remove the API token from the local credential storage:
+
+`cargo logout`
+
+- To add the name of the registry to use:
+
+`cargo logout --registry {{registry}}`
+
+- To use verbose output:
+
+`cargo logout {{-v|--verbose}}`
```

Comment:
While [I actually suggested the syntax you're using here](https://github.com/tldr-pages/tldr/issues/8046#issuecomment-1107658737), we have not yet reached a consensus on how to mention short options.

```suggestion
`cargo logout --verbose`
```